content/foundation/governing-board/index.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Governing Board"
+weight = 3
+extra.summary = """
+The Governing Board is an advisory board that is made up of elected representatives from all across the Matrix ecosystem. The role of the Governing Board is to offer guidance and support to the Guardians, Foundation staff, and Spec Core Team.
+"""
++++
+
+The Governing Board is an advisory board that is made up of elected representatives from all across the Matrix ecosystem. The role of the Governing Board is to offer guidance and support to the Guardians, Foundation staff, and Spec Core Team. It is organized under [these bylaws](/media/2024-04-governing-board-terms-of-reference.pdf) (v1.2, last updated April 2024).
+
+There are nine different constituency groups that are allocated seats on the Governing Board, across three categories:
```

Comment:
there should be a short mention of how to become a member i.e. /support, and maybe to the election page in lieu of a page that explains more of the process.

Reply:
+1 to this. Perhaps:

> To become a member, support the Foundation, be eligible to vote (in the Individual Members category), and even to stand as a candidate, please see [This link]. If you're part of an organisation or community that wants to be part of the Board, please this [this other link] or reach out to us to discuss.

How's that?

Reply:
I applied this with just some minor wording changes as c711057ebb14e8ffb8aabec941df93b8da6d1fd8